pkg/util/kubernetes/hostinfo/node_labels.go

```diff
@@ -0,0 +1,69 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+// +build kubelet,kubeapiserver
```

Comment:
do we need the kubeapiserver flag here? If the cluster agent is enabled we query it instead of the apiserver.

Reply:
i'm wondering if we should decompose it more basically

Reply:
ah, yes, true, it should still compile using the dummy implementation in the apiserver.